Libs/MRML/DisplayableManager/vtkMRMLInteractionEventData.cxx

```diff
@@ -289,6 +290,18 @@ void vtkMRMLInteractionEventData::SetAccuratePicker(vtkCellPicker* picker)
   this->AccuratePicker = picker;
 }
 
+//---------------------------------------------------------------------------
+void vtkMRMLInteractionEventData::SetInteractionContextName(const std::string& v)
```

Comment:
try to avoid single letter variable. It makes it hard to search through the code.

`v` -> `contextName`

Reply:
I was just following the conventions in the file (there are ~10 functions with v as parameter name - I think for "value"). I'll change it to contextName